addons/sale_mrp/tests/test_sale_mrp_kit_bom.py

```diff
@@ -774,3 +774,73 @@ def test_SO_kit_delivery_change_to_comp_packaging(self):
         aggr_prod_qty = so.picking_ids.move_ids.move_line_ids[0]._get_aggregated_product_quantities(kit_name='Kit')
         key = f"{comp_product.id}_{comp_product.display_name}__{comp_product.uom_id.id}_{packaging_comp}_{kit_product.bom_ids.id}"
         self.assertEqual(aggr_prod_qty[key]['packaging_quantity'], 2)
+
+    def test_kit_product_packaging_qty_rounding(self):
+        """
+        Test that when selling a kit and the componenet product is delivered using multiple lots
+        that the product packaging quantity is rounded correctly.
+        """
+        grp_pack = self.env.ref('product.group_stock_packaging')
+        self.env.user.write({'groups_id': [(4, grp_pack.id, 0)]})
```

Comment:
```suggestion
        self.env.user.write({'groups_id': [Command.link(self.ref('product.group_stock_packaging')]})
```
Always use [Commands](https://www.odoo.com/documentation/master/developer/reference/backend/orm.html?highlight=command#odoo.fields.Command) as they're easier to read.